File: Euclid/Util/Extensions/DoubleExtensions.cs
﻿using System;

namespace Euclid.Util.Extensions
{
    public static class DoubleExtensions
    {
        /// <summary>
        /// Convert double for Habbo client
        /// </summary>
        public static string ToClientValue(this double value)
        {
            return String.Format("{0:0.0}", value);
        }
    }
}

File: euclid/src/main/java/euclid/util/extensions/DoubleExtensions.java
package euclid.util.extensions;

import java.util.Locale;

public class DoubleExtensions {

    /**
     * Convert double for Habbo client
     */
    public static String toClientValue(double value) {
        return String.format(Locale.ROOT, "%.1f", value);
    }
}
